### PR TITLE
Add FuzzinessAnalysis category

### PR DESCRIPTION
**Pull request by bsrk**
Require a FuzzinessAnalysis category for fuzzynessanalyis plugin

### CODE REVIEW

File: model/category/category.go [hunk 1]
```diff
@@ -166,7 +172,8 @@ func (c Category) IsFromES() bool {
 
 // IsFromRS checks whether the category is of the reactivesearch category.
 func (c Category) IsFromRS() bool {
-	return c == ReactiveSearch
+	return c == ReactiveSearch ||
+	       c == FuzzinessAnalysis
```
Comment (bietkul) [inline model/category/category.go]: @bsrk Why is that change required?
`FuzzinessAnalysis` is a new category.

You also need to add the limits for that new category.

**Review by bietkul — CHANGES_REQUESTED** [review]

File: model/category/category.go (hunk 1 above)
Comment (bsrk) [reply model/category/category.go]: This plugin does not use olivere's elastic search, but rather uses MakeESRequest in querytranslate.

I'll check and add limits.

Comment (bsrk) [reply model/category/category.go]: I added the limits.

Comment (bsrk) [reply model/category/category.go]: I removed FuzzinessAnalysis from IsFromRS check, though I anticipate this will allow unauthenticated FuzzinessAnalysis request to pass through. Let's resolve this issue if it actually pops ups.

**Review by bsrk — COMMENTED** [review]
I can confirm that there are no changes in plugins/reindexer/*, even though they show up as a diff in this commit.

**Comment by bietkul** [comment]
@bsrk Please resolve conflicts.
